tools/toolchains/mbed_toolchain.py

```diff
@@ -1393,3 +1393,29 @@ def report(self):
         to_ret['linker'] = {'flags': copy(self.flags['ld'])}
         to_ret.update(self.config.report)
         return to_ret
+
+def replace_small_c_lib(target, toolchain):
```

Comment:
Perhaps this should now be called 'should_replace_small_c_lib()'?

Reply:
c57d367